ethcore/src/engines/authority_round/mod.rs

```diff
@@ -24,15 +24,18 @@ use std::sync::atomic::{AtomicUsize, AtomicBool, Ordering as AtomicOrdering};
 use std::sync::{Weak, Arc};
 use std::time::{UNIX_EPOCH, Duration};
 
-use block::*;
-use client::EngineClient;
-use engines::{Engine, Seal, ConstructedVerifier};
+use client_traits::EngineClient;
```

Comment:
I guess the next step would be to move specific engines implementations into it's own crates? ;)

Reply:
You nailed it. :)

Reply:
We're actually already doing that with our WIP Honey Badger engine:
https://github.com/poanetwork/parity-ethereum/tree/hbbft/ethcore/hbbft_engine
Saves a lot of compile time!

Reply:
@afck See https://github.com/paritytech/parity-ethereum/pull/10966 – once you get rid of the `ethcore` dependency compilation times are pretty great.